packages/api/src/parse-cli.ts

```diff
@@ -237,6 +237,11 @@ export default function parseCli(argv?: string | readonly string[]) {
         describe: "google recaptcha secret key",
         type: "string",
       },
+      recaptchaMode: {
+        describe: "google recaptcha mode",
```

Comment:
WDYT of
```suggestion
      enableRecaptcha: {
        describe: "whether to validate user registrations with google recaptcha",
```
Would also need to update places that use it with the new var name.

Alternatively, just a nit: maybe invert the configuration as well and call it `disableRecaptcha` instead, just so the default value is more natural (`false` for a boolean).